### PR TITLE
Move AssertionError to right extension

### PR DESCRIPTION
**Pull request by kukulich**
https://3v4l.org/4N3Ig

### CODE REVIEW

File: Core/Core_c.php [hunk 1]
```diff
@@ -621,7 +612,7 @@ static function bind(Closure $closure, $newthis, $newscope = 'static') { }
      * @since 7.0
      */
     function call ($newThis, ...$parameters) {}
-
```
Comment (wbars) [inline Core/Core_c.php]: Please revert unrelated formatting changes.

Comment (kukulich) [reply Core/Core_c.php]: Done
